thingif/src/main/java/com/kii/thingif/internal/gson/ActionAdapter.java

```diff
@@ -0,0 +1,83 @@
+package com.kii.thingif.internal.gson;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParseException;
+import com.google.gson.JsonPrimitive;
+import com.google.gson.JsonSerializationContext;
+import com.google.gson.JsonSerializer;
+import com.kii.thingif.command.Action;
+
+import java.lang.reflect.Type;
+import java.util.Map;
+
+public class ActionAdapter implements
```

Comment:
Is it necessary that ActionAdapter is public?
Can we make this class package internal?